pkgs/misc/tmux-plugins/default.nix

```diff
@@ -611,12 +611,12 @@ in rec {
   tmux-fzf = mkTmuxPlugin {
     pluginName = "tmux-fzf";
     rtpFilePath = "main.tmux";
-    version = "unstable-2023-07-06";
+    version = "unstable-2023-10-24";
     src = fetchFromGitHub {
       owner = "sainnhe";
       repo = "tmux-fzf";
-      rev = "51081a2688579228d860b3cb410f4437e857fc6e";
-      sha256 = "sha256-qElRHAbnZ+qRasvkfo+lKNahRHklvLOH0BmbQ1oyN6A=";
+      rev = "d62b6865c0e7c956ad1f0396823a6f34cf7452a7";
+      sha256 = "hVkSQYvBXrkXbKc98V9hwwvFp6z7/mX1K4N3N9j4NN4=";
```

Comment:
Could you please use SRI hash instead?
https://nixos.org/manual/nixpkgs/unstable/#fetchurl

Reply:
fixed